src/Compilers/CSharp/Test/Semantic/Semantics/StaticNullChecking.cs

```diff
@@ -14868,7 +14888,7 @@ class Program
 
             comp = CreateStandardCompilation(
                 source,
-                parseOptions: TestOptions.Regular.WithNullCheckingFeature(flags: NullableReferenceFlags.IncludeNonNullWarnings).WithLanguageVersion(LanguageVersion.CSharp7_1));
+                parseOptions: TestOptions.Regular.WithNullCheckingFeature(flags: NullableReferenceFlags.IncludeNonNullWarnings));
```

Comment:
As we discussed, Regular (meaning "default") should probably have all the warnings turned on for the prototype. Removing the `WithNullCheckingFeature` should still work.

Reply:
Thanks, I'll enable non-null warnings (and remove `WithNullCheckingFeature(IncludeNonNullWarnings)`) in a subsequent PR.